src/protected-api-access-protected-api/README.md

```diff
@@ -0,0 +1,208 @@
+
+Use the following settings for your app registration:
+
+| App registration <br/> setting | Value for this sample app                                           | Notes                                                                                                       |
+|:------------------------------:|:--------------------------------------------------------------------|:------------------------------------------------------------------------------------------------------------|
+| **Name**                       | `active-directory-curl-app`                                         | Suggested value for this sample. <br/> You can change the app name at any time.                             |
+| **Supported account types**    | **Accounts in this organizational directory only (Single tenant)**  | Required for this sample. <br/> Support for the Single tenant.                                              |
+| **Platform type**              | **Web**                                                             | Required value for this sample. <br/> Enables the required and optional settings for the app type.          |
+| **API Permissions**            | `api://{clientId}/user_impersonation`                               | Create a new delegated permission called user_impersonation. Required value for this sample.                |
+| **Client secret**              | _Value shown in Azure portal_                                       | :warning: Record this value immediately! <br/> It's shown only _once_ (when you create it).                 |
+
+> :information_source: **Bold text** in the table matches (or is similar to) a UI element in the Azure portal, while `code formatting` indicates a value you enter into a text box or select in the Azure portal.
+
+### 3. Go back to the recently registered web API application
+
+Third, modify the web API application to update the following settings to reference the cUrl app
+
+| App registration <br/> setting    | Value for this sample app                                        | Notes                                                                                              |
+|:---------------------------------:|:-----------------------------------------------------------------|:---------------------------------------------------------------------------------------------------|
+| **knownClientApplications**       | Client ID (UUID) of the application created in step 2.           | Required value for this sample.                                                                    |
+
+### 4. Configure the web API
+
+1. Open the _~/protected-api-access-protected-api/appsettings.json_ file in your code editor and modify the following code:
+
+   ```json
+   "ClientId": "Enter_the_Application_Id_here",
+   "TenantId": "Enter_the_Tenant_Info_Here"
+   "ClientSecret": "Enter_the_Client_Secret_here",
```

Comment:
Does it matter if the right side of these looks different than the right side of them in the code?

Reply:
add a comma at the end as well in the `TenantId` and capitalize the `Here` word.

Reply:
good catch @ckittel,

done | addressed from [773d3](https://github.com/Azure-Samples/ms-identity-docs-code-dotnet/pull/13/commits/773d30d4cdd1a1aa0cbf2446a4bf4ae44b46d241)

Reply:
They are aligned now. Looks good.